### PR TITLE
Add intro section

### PR DESCRIPTION
**Pull request by jwl1997**
Add an intro section(title and paragraph)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # [Sample Repo] PR Practice
 A sample repo for practicing how to create Pull Requests
+#Introduction 
+Creating pull requests(prs) is needed when using RCS in a multi-person projects.
```
Comment (jwl1997) [inline README.md:4]: dscmskjc

**Review by jwl1997 — COMMENTED** [review]
looks good

**Comment by jwl1997** [comment]
k,jnoijol

**Comment by damithc** [comment]
This PR/issue should be against a repo in the org of your module org, not against the repo in se-edu org :-)
Closing...

**Pull request closed without merging**